docs/tempo/website/metrics-generator/app-performance-mgmt.md

```diff
@@ -0,0 +1,167 @@
+---
+aliases:
+- /docs/tempo/latest/server_side_metrics/service_graphs/
+title: Application performance management
+menuTitle: APM dashboard
+weight: 200
+---
+
+# Application performance management dashboard
+
+<span style="background-color:#f3f973;">This is an experimental feature. For more information about how to enable it, continue reading.</span>
+
+Application performance management (APM) uses data gathered from services, agents, systems, and microservices to visualize and identify areas of concern.
+
+The Grafana APM dashboard visualizes the span metrics (traces data for rates, error rates, and durations (RED)) and service graphs.
+Once the requirements are set up, this pre-configured dashboard is immediately available.  
+
+You can use the APM dashboard to:
+
+* Discover spans which are consistently erroring and the rates at which they occur
+* Get an overview of the overall rate of span calls throughout your services
+* Determine how long the top 10% slowest queries in your service take to complete
```

Comment:
```suggestion
* Determine how long the slowest queries in your service take to complete
```

I think in this case maybe it is just easier to be general? Makes the sentence a bit more readable and I figure should you choose you can adjust the percentile to show top X% where X is configurable.

Reply:
Thank you for the suggestion. Much appreciated.